src/hardware/input/mouse_config.cpp

```diff
@@ -198,28 +198,15 @@ static void set_dos_driver_options(const std::string_view options_str)
 		} else {
 			LOG_WARNING(
 			        "MOUSE: Invalid 'builtin_dos_mouse_driver_options' "
-			        "parameter: '%s', ignoring",
+			        "parameter: '%s', using defaults",
 			        token.c_str());
-			config_needs_sync = true;
-		}
-	}
 
-	if (config_needs_sync) {
-		// Config syntax was wrong, create valid setting
-		last_options_str = {};
-		if (mouse_config.dos_driver_immediate) {
-			last_options_str = "immediate";
-		}
-		if (mouse_config.dos_driver_modern) {
-			if (!last_options_str.empty()) {
-				last_options_str += " ";
-			}
-			last_options_str += "modern";
-		}
+			set_section_property_value("mouse",
+			                           "builtin_dos_mouse_driver_options",
+			                           DefaultBuiltinDosMouseDriverOptions);
 
-		set_section_property_value("mouse",
-		                           "builtin_dos_mouse_driver_options",
-		                           last_options_str);
+			set_dos_driver_options(DefaultBuiltinDosMouseDriverOptions);
```

Comment:
It would be nice if we logged the current settings when the users sets these options.

Reply:
OK, added a log.